### DISCUSSION TITLE
Cannot search anything/anywhere

### DISCUSSION THREAD

**Discussion by D-Pow** [community/community#7215] [Mobile] [answered]
On web, GitHub allows searching an entire repository for anything in the top search bar. Within specific files, I can use Ctrl+F to search for terms, or click on a function to see the "Definitions/References" pop-up.

On the mobile app, there are none of these options. I cannot search the repository from the top level nor from a file. I also cannot search from within a file because Ctrl+F doesn't exist. Finally, the "Definitions/References" pop-up doesn't exist either.

I mostly want the search bar to be available everywhere (even if only in the 3 dots menu button) and to be able to search a file in a Ctrl+F like fashion. These are both necessities. The "Definitions/References" pop-up would be nice to have but not nearly as necessary as the other two.

Please add these in! It would greatly enhance the UX and make the app much more usable. Thanks!

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @D-Pow — I've added your note to our tracking issue to help us prioritize this work.

**Reply by deividaspetraitis** [reply]
this feature is essiantal and is the reason why I do not use app on mobile device.

**Reply by koalazub** [reply]
Related: https://github.com/github/feedback/discussions/50

Is there a ticket on the topic already?

**Reply by D-Pow** [reply]
Oh wow, that issue was opened over a year ago! Has there not been any progress at all with search functionality?

**Reply by amrehman** [reply]
Hi @D-Pow , we are tracking this feature request from you and we are keeping an eye out for similar feature requests which will help us prioritise this.

**Reply by Wladefant** [reply]
@amrehman doesn't it make more sense to delete all duplicates and count the votes?
